src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -43,6 +43,20 @@
 
 logger = getLogger(__name__)
 
+# We allow some common aliases to be used
+OPTION_ALIASES = {
+    "HEADER": ("SKIP_HEADER", lambda val: 1 if val else 0),
+    "DELIMITER": ("FIELD_DELIMITER", lambda val: val),
+    "SEP": ("FIELD_DELIMITER", lambda val: val),
+    "LINESEP": ("RECORD_DELIMITER", lambda val: val),
```

Comment:
I'm a bit concerned about the `LINESEP` since if the RECORD_DELIMITER is not a new line delimiter (aka `\n`), there's no concept of a line.